### PR TITLE
feat: support secrets 

### PR DESCRIPTION
**Pull request by zac-li**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -369,7 +370,120 @@ lc-serve deploy jcloud --app endpoints:app
...
+```bash
+lcserve deploy jcloud app --secrets .env
+```
+
+<details>
+<summary>Show details</summary>
+
+Let's take an example of a simple FastAPI app that uses an external Redis instance for rate-limiting endpoints.
+
+### 🚦 Deploy a FastAPI app with redis based rate-limiting
```
Comment (deepankarm) [inline README.md:386]: Can we please change this to an example that uses `OPENAI_API_KEY` instead of redis?

Comment (zac-li) [reply README.md:386]: sure

Comment (zac-li) [reply README.md]: ok revised

**Review by deepankarm — CHANGES_REQUESTED** [review]
Please wait until we release new jcloud version that supports secrets.

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 jina==3.15.2
 click
-jcloud>=0.2.8,<=0.2.12
+jcloud
```
Comment (deepankarm) [inline requirements.txt:3]:
```suggestion
jcloud>=0.2.13
```

**Review by deepankarm — CHANGES_REQUESTED** [review]

**Review by deepankarm — APPROVED** [review]

**Pull request merged**